FrejaEidClient/src/main/java/com/verisec/frejaeid/client/beans/general/OrganisationIdInfo.java

```diff
@@ -14,23 +15,28 @@ public class OrganisationIdInfo {
     private final String identifier;
     private final Map<String, String> issuerFriendlyName;
     private final String issuerCode;
+    private final List<OrganisationIdAttribute> additionalAttributes;
 
     /**
      * Creates instance of {@linkplain OrganisationIdInfo}.
      *
-     * @param identifier         User's Organisation ID. It cannot be {@code null} or empty.
-     * @param issuerFriendlyName Friendly name of the organisation which issued the user's Organisation ID
-     *                           in Swedish and English. It cannot be {@code null} or empty.
-     * @param issuerCode         Unique identifier of the organisation which issued the user's Organisation ID. It
-     *                           can be {@code null} or empty.
+     * @param identifier           User's Organisation ID. It cannot be {@code null} or empty.
+     * @param issuerFriendlyName   Friendly name of the organisation which issued the user's Organisation ID
+     *                             in Swedish and English. It cannot be {@code null} or empty.
+     * @param issuerCode           Unique identifier of the organisation which issued the user's Organisation ID. It
+     *                             can be {@code null} or empty.
+     * @param additionalAttributes additional attributes related to the identifier.
```

Comment:
**[MINOR]**
The limit of maximum number of attributes that can be set is 10. It might be good to mention it here.
But then again, for other fields we don't mention limitations (e.g the maximum length of identifier is not mentioned here.

Reply:
Yes, I think that sort of limit is usually only described in documentation, not the javadoc.